base_asset/behavior/data_rest.py

```diff
@@ -127,5 +128,12 @@ def import_batch(self):
     return results
 
   def get_ignored_properties(self):
-    return ['createdBy', 'createdAt', 'lastModifiedBy', 'lastModifiedAt', 'activationCode', 'ACTION', 'lumavateId', 'namespace','orgId']
+    excluded_properties = ['createdBy', 'createdAt', 'lastModifiedBy', 'lastModifiedAt', 'activationCode', 'ACTION', 'lumavateId', 'namespace','orgId']
+
+    if self.is_batch_request:
+      column_defs = self._model_class.get_column_definitions(self._asset_id)
+      file_columns = [ column['columnName'] for column in column_defs if column['columnType']['value'] == 'file']
```

Comment:
This should utilize the column type enum
```suggestion
      file_columns = [ column['columnName'] for column in column_defs if column['columnType']['value'] == enums.ColumnDataType.FILE.value]
```

Not sure if the .value part is needed or if it will automatically evaluate to str.